crates/witness-generator/README.md

```diff
@@ -35,37 +55,40 @@ witness-generator = { path = "../witness-generator" } # Adjust path as needed
 
 Example (conceptual):
 
-```rust
-// use witness_generator::generate_stateless_witness::generate;
-use witness_generator::BlocksAndWitnesses;
+```rust,no_run
```

Comment:
`no_run` since now writes output to folders, so isn't a nicely runnable example.
Open to feedback if there's a better way -- I didn't want to "comment" stuff since sounds like cheating :P

Reply:
We can remove it and make an issue to create an integration example in the future -- if its no_run, then its likely that it will be outdated in the future without us knowing

Reply:
https://github.com/eth-act/zkevm-benchmark-workload/issues/96